applications/volume_rendering_xr/Dockerfile

```diff
@@ -91,6 +83,49 @@ RUN apt update \
         libxxf86vm-dev \
         mesa-common-dev \
     && rm -rf /var/lib/apt/lists/*
+    
+# Install Monado OpenXR runtime. See
```

Comment:
Noting: With the recent update you purely rely on our runtime (Which is based on Monado itself), and don't pull in Monado directly